zipline/src/commonMain/kotlin/app/cash/zipline/internal/bridge/outbound.kt

```diff
@@ -33,6 +33,7 @@ internal interface OutboundBridge {
     @PublishedApi internal val endpoint: Endpoint,
   ) {
     val serializersModule = json.serializersModule
+    var closed = false
```

Comment:
We don't define that one in IR, that's cool?

Reply:
I don’t follow?

Reply:
We added code in IR to do `context.close = true`, don't we need IR code that add the line `var closed = false` ?